Resources/Prototypes/Roles/Jobs/departments.yml

```
@@ -169,3 +169,29 @@
+  id: DepGerman
+  name: department-German
+  description: department-German-description
+  color: "#080101"
+  roles:
+    - GermanCaptain
+    - GermanSergeant
+    - GermanSubmachineGunner
+    - GermanRifleman
+    - GermanMachinegunner
+    - GermanMedic
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)